python/ray/exceptions.py

```diff
@@ -747,6 +747,13 @@ class ObjectRefStreamEndOfStreamError(RayError):
     pass
 
 
+@DeveloperAPI
+class ActorUnavailableError(RayError):
+    """Raised when the actor is temporarily unavailable but may be available later."""
```

Comment:
I am a little concerned that such a cause may not be accurate… like a connection broken actor may turn to restart

Reply:
Is this the only case where the cause may not be accurate: an actor is both restarting and the network has high latency. The `ActorUnavailableError` reports that it was caused by high latency, but it was actually due to both? If this is the only case, then I think it's okay because the `ActorUnavailableError` was still actually caused by high latency.

Or are there more cases? E.g. the network is fine and the actor is restarting, but the error reports that it was caused by high latency?

As a side note, this is also why I'm curious why we decided to combine these cases into one error type. Why not  separate it into `NetworkUnavailableError` and `ActorRestartingError`?